### ISSUE TITLE
Warnings about css files not in allowed paths

### ISSUE THREAD

**Issue by EnzephaloN** [quellenform/t3x-iconpack-fontawesome#1]
Hello there
I've installed tx-iconpack, tx-iconpack-fontawesome and tx-bootstrap-package-iconpack in my TYPO3 12.4.20.

All seems to work. Until I looked into system-log, where I foumnd lots of warnings about css-files of iconpack.

````
Core: Error handler (BE): PHP Warning: file_exists(): open_basedir restriction in effect. File(/
_assets/9bf999188554c80cf998670bee47d3cf/Vendor/fontawesome-free-6.5.1-web/css/brands.min.css) is not within the allowed 
path(s): (/var/www/clients/web:/var/www/clients//private:/var/www/clients//tmp:/var/www/www.adresse.com/web:/srv/www/
www.adresse.com/web:/usr/share/php5:/usr/share/php:/usr/share/phpmyadmin:/etc/phpmyadmin:/var/lib/phpmyadmin:/dev/
random:/dev/urandom) in /var/www/clients/web/adresse/vendor/typo3/cms-core/Classes/Utility/GeneralUtility.php line 2181
``````
It looks like every min.css-file has such a warning.

These are the only one I have. All other extensions assets do not throw warnings.

Maybe it's because the assets-paths begins with an "/" ??

Best regards
EnzephaloN

**Comment by stephankellermayr** [comment]
Unfortunately, I cannot reproduce the error you mentioned.

And since I assume that the backend otherwise works correctly and all other CSS files can be loaded, it is strange that this file of all files cannot be loaded.

Please check whether the extension is really fully installed.

It would be helpful to know:
- Which extension versions are you using here?
- Does everything else in the backend work?
- Did you install the extensions via TER or via composer?

**Comment by stephankellermayr** [comment]
Strangely, at first glance the situation does not look unusual.

However, as I can't reproduce the problem, I can't say much about it at the moment.

Can the file be opened via a direct link in the browser?

**Comment by EnzephaloN** [comment]
> Can the file be opened via a direct link in the browser?

Yes. If I copy the address into a browser, pasted behind www.address.com, I can open those files.

**Comment by EnzephaloN** [comment]
Its not a fix, but I get no more entries in log:
I set [SYS][belogErrorReporting] = 4096
